static/src/javascripts/projects/common/modules/navigation/user-account.js

```diff
@@ -50,8 +56,13 @@ const showMyAccountIfNecessary = (): void => {
                     });
 
                     accountActionsLists.forEach(accountActions => {
-                        accountActions.classList.remove('u-h');
+                        accountActions.classList.remove('is-hidden');
                     });
+
+                    // We still want the button to be hidden, but tabbable now
+                    if (accountTrigger) {
+                        accountTrigger.classList.toggle('is-hidden', 'u-h');
```

Comment:
`toggle`'s second argument is a boolean that forces the class to be added if true, or removes it if false. To toggle multiple classes, you might need to pass the class names as a single string:
```js
accountTrigger.classList.toggle('is-hidden u-h');
```

Reply:
I just realised that too and pushed up a change! Don't know what I was thinking the first time around.